packages/pwa/app/utils/locale.js

```diff
@@ -48,8 +48,11 @@ export const loadLocaleData = async (locale) => {
  */
 export const getLocaleConfig = async ({getUserPreferredLocales} = {}) => {
     const preferredLocales = getUserPreferredLocales ? getUserPreferredLocales() : [DEFAULT_LOCALE]
-    const targetLocale = getTargetLocale(preferredLocales, SUPPORTED_LOCALES, DEFAULT_LOCALE)
-    const messages = await loadLocaleData(targetLocale)
+    const targetLocale = whichLocaleToLoad(preferredLocales, SUPPORTED_LOCALES, DEFAULT_LOCALE)
+
+    const messages = await loadLocaleData(
+        typeof window === 'undefined' ? process.env.PSEUDOLOCALE || targetLocale : targetLocale
```

Comment:
Pseudo locale is now isolated to this part of the code only. This way the app could still have the target locale being English, French, etc. while the actual translations being dynamically loaded to use the pseudo locale.